components/plan-build/bin/hab-plan-build.sh

```diff
@@ -1603,6 +1611,22 @@ _resolve_dependencies() {
   _validate_deps
 }
 
+# **Internal** Loads specified scaffolding to extend plan DSL. This assumes
+# the scaffolding lives within `lib` and is named `scaffolding.sh` for each
+# application/project type.
+_load_scaffolding() {
+  local scaffolding="${pkg_scaffolding:-}"
+
+  if [[ -n "$scaffolding" ]]; then
+    build_line "Loading Scaffolding $scaffolding"
+    if source "$(pkg_path_for $scaffolding)/lib/scaffolding.sh"; then
+      build_line "Scaffolding loaded"
+    else
+      exit_with "Failed to load Scaffolding $scaffolding" 1
```

Comment:
It's true that we failed to load, but why? It would be nice to provide a helpful message that `lib/scaffolding.sh` was not found in the root of the package